### PR TITLE
Recordings Page

### PR DESCRIPTION
**Pull request by ghostdevv**
Closes MKWEB-76

@chrifro do you have a list of all the recordings? I need the following information for each:

- Title
- Description
- Person's Name
- Person's Job title
- Link to recording

<a href="https://gitpod.io/#https://github.com/gitpod-io/CDE-Universe/pull/156"></a>

### CODE REVIEW

**Comment by chrifro** [comment]
The [Figma mockup](https://www.figma.com/file/Gm3Fs2yADw6xFrmJkeXk1V/CDE-universe?type=design&node-id=558%3A1713&mode=design&t=368RrKGGVv51TJeD-1) should already include all needed speaker information. 

@pawlean could you confirm and add any missing information?  🙏

**Comment by ghostdevv** [comment]
I don't see any links for the talks?

**Comment by chrifro** [comment]
You can view the YouTube playlist [here](https://www.youtube.com/playlist?list=PL3TSF5whlprWDA0q5GshRk-yPRGJzaY3q). @pawlean could you help with anything relating to content? 🙏

**Comment by ghostdevv** [comment]
Updated the content

**Review by Siddhant-K-code — COMMENTED** [review]
- [x] 💭 **Suggestion (non-blocker)**: We could align these designs (fixed width & centred text of description of recordings).

image

---

- [x] 💭 **Suggestion (blocker)**: Update homepage <kbd>View recordings</kbd> buttons to `/recordings`. Currently, they are redirecting to the YouTube playlist.

---

- [ ] 💭 **Suggestion**: Update Meta Title, Description, Keywords using `OpenGraph` component something like this:

https://github.com/gitpod-io/CDE-Universe/blob/251e435de99b72427b5fe575f072242c0194305e/src/routes/%2Bpage.svelte#L14-L22

Suggestions for Meta tags:

- [x] *(blocker)* **Title**: `CDE Universe 2023 Talk Recordings | Learn from the experts`
- [x] *(non-blocker)* **Description**: `Watch all the talk recordings from CDE Universe 2023. This includes keynote speakers, breakout sessions, and more. Learn from the experts and get inspired to start your CDE Journey today `
- [x] *(non-blocker)* **Keywords**: All the keywords above including `recordings, conference recordings, talk recordings`

cc: @chrifro / @taliamoyal for Meta Title, Keywords suggestions.

**Comment by ghostdevv** [comment]
>  Suggestion (non-blocker): We could align these designs (fixed width & centred text of description of recordings).

This is a bug, fixed!

File: src/routes/recordings/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    },
+  ];
+</script>
+
+<OpenGraph
+  data={{
+    title: "CDE Universe 2023 Talk Recordings | Learn from the experts",
+    description:
+      "Watch all the talk recordings from CDE Universe 2023. This includes keynote speakers, breakout sessions, and more. Learn from the experts and get inspired to start your CDE Journey today ",
+    keywords: "recordings, conference recordings, talk recordings",
```
Comment (Siddhant-K-code) [inline src/routes/recordings/+page.svelte:70]:
```suggestion
    keywords: "CDE Universe recordings, conference recordings, talk recordings, CDE Universe talks",
```

**Review by Siddhant-K-code — APPROVED** [review]
LGTM ✅

**Review by chrifro — APPROVED** [review]
Looks great, thank you both 😊

Note: we might need to have a follow-up PR with additional videos, I'll ping Pauline for more info on that

**Pull request merged**